Tests/Docker/Container/LoggerTest.php

```diff
@@ -597,8 +597,7 @@ function (Event $event) use (&$contents) {
             $container->run();
         } catch (UserException $e) {
             self::assertStringContainsString('message to stderr', $e->getMessage());
-            self::assertGreaterThan(4000, strlen($e->getMessage()));
-            self::assertLessThan(4050, strlen($e->getMessage()));
+            self::assertSame(3999, strlen($e->getMessage()));
```

Comment:
Předtím to mělo 4000 znaků + " ... ". Ten nový helper má max 4000 znaků a počítá do toho i " ... ". Takže to pak vychází takhle